src/webgpu/shader/execution/expression/call/builtin/texture_utils.spec.ts

```diff
@@ -0,0 +1,108 @@
+export const description = `
+Tests for texture_utils.ts
+`;
+
+import { makeTestGroup } from '../../../../../../common/framework/test_group.js';
+import { assert } from '../../../../../../common/util/util.js';
+import { GPUTest } from '../../../../../gpu_test.js';
+import { getTextureDimensionFromView, virtualMipSize } from '../../../../../util/texture/base.js';
+import {
+  kTexelRepresentationInfo,
+  PerTexelComponent,
+  TexelRepresentationInfo,
+} from '../../../../../util/texture/texel_data.js';
+
+import {
+  chooseTextureSize,
+  createTextureWithRandomDataAndGetTexels,
+  isSupportedViewFormatCombo,
+  readTextureToTexelViews,
+  texelsApproximatelyEqual,
+} from './texture_utils.js';
+
+export const g = makeTestGroup(GPUTest);
+
+function texelFormat(texel: Readonly<PerTexelComponent<number>>, rep: TexelRepresentationInfo) {
+  return rep.componentOrder.map(component => `${component}: ${texel[component]}`).join(', ');
+}
+
+g.test('readTextureToTexelViews')
+  .desc('test readTextureToTexelViews for various formats and dimensions')
+  .params(u =>
+    u
+      .combineWithParams([
+        { srcFormat: 'r8unorm', texelViewFormat: 'rgba32float' },
+        { srcFormat: 'r8sint', texelViewFormat: 'rgba32sint' },
+        { srcFormat: 'r8uint', texelViewFormat: 'rgba32uint' },
+        { srcFormat: 'rgba32float', texelViewFormat: 'rgba32float' },
+        { srcFormat: 'rgba32uint', texelViewFormat: 'rgba32uint' },
+        { srcFormat: 'rgba32sint', texelViewFormat: 'rgba32sint' },
+        { srcFormat: 'depth24plus', texelViewFormat: 'rgba32float' },
```

Comment:
Can we add a `{ srcFormat: 'depth24plus', texelViewFormat: 'r32float' },`

Reply:
added